workspaces/spectacle/src/in-memory/index.ts

```diff
@@ -261,7 +261,7 @@ export class InMemoryDiffService implements IOpticDiffService {
         this.dependencies.opticEngine.learn_undocumented_bodies(
           spec,
           interactionsJsonl,
-          'sequential'
+          'random'
```

Comment:
@devdoshi  -- Whenever we document new endpoints in an example session they are getting ShapeID conflicts. Is changing this right? Can it be parameterized?

Reply:
They shouldn't get conflicts with the random one and it should be random by default. For the sequential one we would some extra state